e2e/page-objects/addDetailPage.js

```diff
@@ -0,0 +1,102 @@
+        .assert.containsText('@assertComment','Add Comment')
+    },
+    addCorrection:function(browser){
+        return this.waitForElementVisible('@chevronForCorrection')
+        .click('@chevronForCorrection')
+        .waitForElementVisible('@fileCorrection')
+        .click('@fileCorrection')
+    },
+    correctionFiling:function(browser){
+        return this
+        .assert.containsText('@correctionHeader','Correction — Annual Report (2019)')
+        .assert.containsText('@dateAssertion','Original Filing Date: 2019-03-05')
+        .assert.containsText('#correction-step-1-header','1. Detail')
+        .setValue('#detail-comment-textarea','Filed Correction')
+        .assert.containsText('#correction-step-2-header','2. Certify')
+        .setValue('#certified-by-textfield','Test')
+        .click('@certifyCheckbox')
+    },
+
+    completePyament:function(routingslip){
+        return this
+        .assert.containsText('#correction-step-3-header','3. Staff Payment')
+        .setValue('#routing-slip-number-textfield',routingslip)
+        .click('@prioritycheckbox')
+    },
+    checkFeeByIndex: function (feeName, feeValue, desiredIndex) {
+        return this
+            .assert.containsText({selector: '@feeName', index: desiredIndex}, feeName)
+            .assert.containsText({selector: '@feeValue', index: desiredIndex}, feeValue);
+    },
+    checkFeeCount: function (expectedCount) {
+        return this.expect.elements('@feeListItem').count.to.equal(expectedCount);
+    },
+    checkTotalFees: function (expectedTotalString) {
+        return this.assert.containsText('@feeTotal', expectedTotalString);
+    },
+    dashboardAssertion: function(browser){
+        return this
+        .assert.containsText('@todoTitle','To Do (2)')
+        .assert.attributeEquals('@addressChangeButton','disabled', 'true')
+        .assert.attributeEquals('#standalone-directors-button','disabled', 'true')
+    }
+
+};
+module.exports = {
+    commands: [addDetailsCommands],
+    elements:{
+    chevron:{
+        selector:"div:nth-child(1)  button i",
+        index:5
+    },
+    addDetailButton:{
+    selector:' div.v-menu__content.theme--light.menuable__content__active  div:nth-child(2)',
+    index:1
+    },
+    addDetailDialogBox:"#dialog-title",
+    addDetailArea:"#detail-comment-textarea",
+    saveButton:"#dialog-save-button",
+    addedDetailButton:{
+            selector:" div:nth-child(1) > button > div.list-item",
+            index:1
+    },
+    assertDetail:"#details-list  h4",
+    assertComment:'div:nth-child(3)  div.v-list-item__subtitle.body-2  div',
+    fileCorrection:{
+        selector:" div.v-menu__content.theme--light.menuable__content__active  div:nth-child(1)",
+        index:2
+    },
+    chevronForCorrection:{
+        selector:"div:nth-child(1)  button i",
+        index:6
+    },
+    correctionHeader:"#correction-header",
+    dateAssertion:"#correction-article > header > p",
```

Comment:
Try #correction-header + p

The + means adjacent sibling. (on the same level, not nested below)